openmdao/utils/assert_utils.py

```diff
@@ -151,7 +155,8 @@ def assert_check_partials(data, atol=1e-6, rtol=1e-6):
                             norm_string = '{}'.format(norm)
                             err_msg = '{0} | {1} | {2} | {3}'.format(
                                 pad_name(wrt_string, len_wrt_width),
-                                pad_name(error_type.split()[0], len_absrel_width),
+                                pad_name(error_type.split()[
+                                         0], len_absrel_width),
```

Comment:
Not sure why the split is so weird here, but we have agreed for this project to allow 100-character lines, and have adjusted our linting as such.